src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  return (
```

Comment:
use destructuring from the movie
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
  const { ... } = movie;
  return (
```

Reply:
Done